templates/learn.html

```diff
@@ -15,6 +15,19 @@
       </div>
     </div>
   </a>
+  <a class="card" href="/learn/migration-guides/index">
```

Comment:
```suggestion
  <a class="card" href="/learn/migration-guides/introduction">
```

Without this we get a 404 status code when clicking this card